docs/source/formulations_example.rst

```diff
@@ -0,0 +1,648 @@
+.. formulations_example:
+
+Formulations Example
+====================
+
+The Citrine Platform is a powerful tool to analyze formulations data.
+A formulation is, generally speaking, a mixture of mixtures.
+Individual ingredients are mixed together to form more complex ingredients, which may themselves be mixed with other ingredients and/or mixtures, and so on.
+Generally, the top-level formulation has some properties that we with to optimize.
+The ingredients themselves may also have known properties, which are useful for understanding the properties of the top-level formulation.
+In addition, each mixing process may be characterized by attributes such as the temperature of a reaction vessel or the speed of a mixing element.
+
+The GEMD data model provides a way to record all of the information relevant to a formulations problem,
+and Citrine's AI Engine turns this information into trained models that can predict the properties of novel formulations.
+This section demonstrates how to express a complete formulations example in Citrine-python, from data ingestion to new candidate generation.
+
+Example Raw Data
+----------------
+
+To demonstrate the formulations machinery, we consider a toy problem -- margaritas!
+We scour our pantry and gather some ingredients, laid out in the table below.
+Several ingredients have labels, which helps the Citrine Platform understand the relationships between ingredients.
+We have also measured the sugar fraction for some of the ingredients, and we know the price per kilogram of each ingredient.
```

Comment:
MAYBE - Could we use something else besides $/kg? This is relevant to Materials as well, but we don't support this unit natively in GEMD (AFAIK) and may set poor expectations.

Reply:
Concur.  Perhaps trying to hit a particular alcohol content?

Reply:
In the actual example, I use `1/kg` as the unit.

Constraining on a mean property is such a common use case that it should be demonstrated, and limiting price seems to be the most common manifestation. But if you think that the unit issue is too thorny, then I agree that "alcohol content" is another example that could demonstrate the concept.

Reply:
I added a note about the currency unit